Deploy/configparser.cpp

```diff
@@ -1107,26 +1113,30 @@ bool ConfigParser::initQmake() {
                 auto env = QProcessEnvironment::systemEnvironment();
                 auto proc = DeployCore::findProcess(env.value("PATH"), "qmake");
                 if (proc.isEmpty()) {
-                    QuasarAppUtils::Params::log("The deployment target requires Qt libs, but init qmake is failed."
-                                                "Use the qmake option for set a path to qmake.",
+                    QuasarAppUtils::Params::log("The deployment target requires Qt libraries,"
+                                                " but initialize of Qt directories is failed."
+                                                " Please use the qmake option to set a path to the qmake executable.",
                                                 QuasarAppUtils::Error);
                     return false;
                 }
 
                 return initQmakePrivate(proc);
             }
 
-            QuasarAppUtils::Params::log("You Application requires Qt libs but qmake cannot be found in option 'qmake' or PATH."
-                                        "You use option noCheckPATH. Disable the option noCheckPATH from your deploy command for search qmake from PATH",
+            QuasarAppUtils::Params::log("Your distribution required Qt libraries "
+                                        "but qmake executable cannot be found in the 'qmake' option or PATH. "
+                                        "You are using the option noCheckPATH, "
```

Comment:
```suggestion
                                        "but qmake executable cannot be found in the 'qmake' option or RPATH. "
                                        "You are using the option noCheckPATH, "
```

Reply:
Agree. Reasons:

1. If `noCheckPATH` is enabled, we'll never try to initialize Qt by searching qmake in `PATH` .
2. `getQtPathesFromTargets()` will analyze `RPATH` of targets. If nothing is found and `noCheckPATH` is enabled, we will print this error message, so its content should match the actual situation.

Ref: https://github.com/QuasarApp/CQtDeployer/pull/578#discussion_r624593687 in https://github.com/QuasarApp/CQtDeployer/pull/578#pullrequestreview-646698110

Reply:
We check first time qmake option if the qmake otion is failed we check RPATH and if the RPATH failed we check noCheckPATH option and if the noCheckPATH is enabled we print this message.